src/sentry/web/frontend/integration_setup.py

```diff
@@ -3,15 +3,17 @@
 import logging
 
 from sentry.integrations.pipeline import IntegrationPipeline
-from sentry.web.frontend.base import BaseView
+from sentry.web.frontend.base import OrganizationView
```

Comment:
actually -- is organization slug even in the url path?

Reply:
No actually. but afaict the OrganizationView pulls this from somewhere else. Again it's not strictly needed here, but it is consistent.

Reply:
it might pull from session when it can, but i think it might error if it cant

tl;dr integration_setup has no organization present in some cases, which is why this didnt use OrganizationView. I think you are going to have to somehow do this in the pipeline, since the pipeline is where it fetches the organization access from.

Reply:
Nah this is fine, I'll just switch it back in that case. It didn't really need it, I kind of just wanted to this endpoint to be consistent. But yeah, you're right setup doesn't have the org.

Reply:
4b3fba1